internal/providers/sdkv2/tests/coverage_test.go

```diff
@@ -1,4 +1,4 @@
-package provider
+package tests
```

Comment:
I haven't checked much what these tests are actually testing. Though, if they are testing `sdkv2/provider.go` then move them in `sdkv2` next to the code they are testing.

Reply:
These don't test the provider.go. They check for whether the right stub is generated for unit tests and check for coverage of resources by creating completeness.md. 

One of the tests is skipped and I think these are probably dead code so need to confirm if they are needed or not. Since they are not testing provider.go (now sdkv2.go), so not moving them there.